### ISSUE TITLE
`make frontend` broken on Debian 10

### ISSUE THREAD

**Issue by vext01** [buildbot/buildbot#5033]
```
$ make frontend
...
Requirement already satisfied: cffi!=1.11.3,>=1.8 in ./.venv/lib/python3.7/site-packages (from cryptography>=2.7->autobahn>=0.16.0->buildbot) (1.12.3)
Requirement already satisfied: asn1crypto>=0.21.0 in ./.venv/lib/python3.7/site-packages (from cryptography>=2.7->autobahn>=0.16.0->buildbot) (1.0.1)
Requirement already satisfied: pycparser in ./.venv/lib/python3.7/site-packages (from cffi!=1.11.3,>=1.8->cryptography>=2.7->autobahn>=0.16.0->buildbot) (2.19)
cd www/build_common; yarn install --pure-lockfile
Usage: yarn [options]

yarn: error: no such option: --pure-lockfile
make: *** [Makefile:47: frontend_deps] Error 2
```

There are two pieces of software called "yarn" and we are using the wrong one.

I notice the following when installing the yarn pkg:
```
$ sudo apt-get install yarn
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'cmdtest' instead of 'yarn'
...
```

If it's the nodejs `yarn` we need, that's the `yarnpkg` package:
```
$ apt-cache search yarnpkg
yarnpkg - Fast, reliable and secure npm alternative
```

As a workaround, I made a symlink from `/usr/bin/yarnpkg` to `yarnpkg` in the python virtualenv and it works fine.

**Comment by kpweiler** [comment]
This is also true for Ubuntu 18.04 meaning the documentation here:

http://docs.buildbot.net/current/developer/quickstart.html#prerequisites
> The below method has been tested on Ubuntu 18.04 and should work on recent enough Debian.

is wrong

**Comment by p12tic** [comment]
This bug has been fixed in fa7811d44b4a5617911b89839af4ebc377595919.
